packages/blockchain-link-types/src/blockfrost.ts

```diff
@@ -169,6 +169,22 @@ export interface BlockfrostAccountInfo {
         total: number;
         index: number;
     };
+    misc: {
+        staking: {
+            address: string;
+            isActive: boolean;
+            rewards: string;
+            poolId: string | null;
+            drep: {
+                drep_id: string;
+                hex: string;
+                amount: string;
+                active: boolean;
+                active_epoch: number | null;
+                has_script: boolean;
```

Comment:
```suggestion
            drep: {
                drep_id: string;
                hex: string;
                active: boolean;
                has_script: boolean;
```

maybe we don't need all the data? we could even omit `active`, but I feel like that could be useful in the future :D